std/src/main/scala/cats/std/anyval.scala

```diff
@@ -0,0 +1,94 @@
+package cats
+package std
+
+trait AnyValInstances
+  extends IntInstances
+  with    ByteInstances
+  with    CharInstances
+  with    LongInstances
+  with    ShortInstances
+  with    FloatInstances
+  with    DoubleInstances
+  with    BooleanInstances
+  with    UnitInstances
+  with    BigIntInstances
```

Comment:
```BigInt``` is an ```AnyRef```, but I could see how you might want to include it. One could probably make a similar argument for ```BigDecimal```. http://www.scala-lang.org/api/current/index.html#scala.math.BigInt